client/login/wp-login/components/one-login-footer.scss

```diff
@@ -1,20 +1,15 @@
 .one-login__footer {
     display: flex;
-    flex-direction: column;
```

Comment:
I think this change broke the alignment in the main login form footer e.g. where we show `<- Back to [client]` for the one or two cases that we do. They should be stacked:

| Before | After/Now |
| --- | --- |
| <img width="600" alt="Screenshot 2025-08-26 at 4 19 16 PM" src="https://github.com/user-attachments/assets/fcd8a8a6-1c9f-4509-9f7a-0bd5c7f72430" /> | <img width="600" alt="Screenshot 2025-08-26 at 4 18 09 PM" src="https://github.com/user-attachments/assets/74723fbf-3d9c-46be-b951-06099f1b25e8" /> |

I'll work it out / fix in https://github.com/Automattic/wp-calypso/pull/104891